diff --git a/Makefile.rules_generic b/Makefile.rules_generic
index 4ea9574..9047280 100644
--- a/Makefile.rules_generic
+++ b/Makefile.rules_generic
@@ -34,9 +34,9 @@ endif
 
 ifeq ($(shell $(LD) -v 2>&1 | grep clang),)
 LD := $(CC)
-ifneq ($(shell [ `clang -v 2>&1 | grep "version " | cut -f3 -d' ' | cut -f1 -d'.'` -ge 7 ] && echo ok),ok)
-$(error Requires at least CLANG 7 to link correctly with -fropi -frwpi)
-endif
+#ifneq ($(shell [ `clang -v 2>&1 | grep "version " | cut -f3 -d' ' | cut -f1 -d'.'` -ge 7 ] && echo ok),ok)
+#$(error Requires at least CLANG 7 to link correctly with -fropi -frwpi)
+#endif
 $(info Linker changed to CLANG)
 endif
 
